x-pack/plugins/spaces/public/views/nav_control/nav_control_popover.tsx

```diff
@@ -79,6 +79,8 @@ export class NavControlPopover extends Component<Props, State> {
         closePopover={this.closeSpaceSelector}
         anchorPosition={'rightCenter'}
         panelPaddingSize="none"
+        // @ts-ignore
+        repositionOnScroll={true}
```

Comment:
FWIW, you can just set `repositionOnScroll` without passing in `={true}` and it will default to true.